python/setup.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+
+import toml
+from setuptools import setup
+from setuptools_rust import Binding, RustExtension
+
+
+CUR_DIR = Path(__file__).resolve().parent
+
+# python package version is taken from Cargo.toml to avoid duplication
+with open(str(CUR_DIR / "Cargo.toml"), "r", encoding="utf-8") as f:
+    cargo = f.read()
+    cargo_toml = toml.loads(cargo)
+    cargo_version = cargo_toml['package']['version']
+
+with open(str(CUR_DIR / "README.md"), "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+with open(str(CUR_DIR / "requirements.txt")) as f:
+    requirements = f.read().splitlines()
+
+setup(
+    name="microvmi",
+    version=cargo_version,
+    author="Mathieu Tarral",
```

Comment:
There is a lot of duplicated information here. Can't you somehow funnel the information from Cargo.toml to setuptools?

Reply:
with the `toml` python package, we are already forward the version information.
good idea, let's do it for the rest of the setuptools fields

Reply:
Did you forget about this?

Reply:
i could parse the Cargo author field to fill this one